core/src/components/item-sliding/item-sliding.tsx

```diff
@@ -116,6 +116,24 @@ export class ItemSliding implements ComponentInterface {
     return Promise.resolve(this.getSlidingRatioSync());
   }
 
+  /**
+   * Open the sliding item.
+   */
+  @Method()
+  async open() {
+    if (this.openAmount !== 0 && this.item !== null) {
+      const options = this.el.querySelector('ion-item-options');
+      if (options) {
+        this.item.style.transition = 'transform 500ms cubic-bezier(0.36, 0.66, 0.04, 1)';
```

Comment:
You shouldn't need to manually apply this transition style here. Try setting `this.state = SlidingState.Enabled`. That _should_ fix it

Reply:
Will make this change & test it out tonight.

Reply:
This change didn't work. :(

Reply:
I will keep the transition style change as it is since it is working. Please let me know if that's okay?